check.go

```diff
@@ -74,7 +74,7 @@ func Check(c *gin.Context) {
 
 		chlng, err := hex.DecodeString(c.PostForm("chlng"))
 		if err != nil {
-			utilities.LogError(ravenClient, "Unable to decode chlng string", err)
+			utilities.LogError(ravenClient, "Unable to decode chlng", err)
```

Comment:
This would be more cosmetic - the issue truly is about decoding the string itself, not the chlng.